ReadMe.md

```diff
@@ -94,15 +97,20 @@ to configure it accordingly:
   of `Array.copy` calls in the background which are slow.
 - Use `system` to add a system to your world. The order of `system` calls defines the order in which they are called
   when calling `world.update`
+- Use `component` to register components and its component listeners (if needed and available, see below) to your world.
 
 Here is an example that creates a world for 1000 entities with a Move- and PhysicSystem:
 
 ```Kotlin
 val world = World {
     entityCapacity = 1000
 
-    system<MoveSystem>()
-    system<PhysicSystem>()
+    system(::MoveSystem)
+    system(::PhysicSystem)
+
+    component(::Position)
+    component(::Physic)
+    component(::Box2dComponent, ::Box2dComponentListener)
```

Comment:
I like that change a lot because in my opinion a listener is always for one specific component and registering it directly with the Component is a cool idea. I will take that over to the master as well.